store/encoding.go

```diff
@@ -0,0 +1,167 @@
+package store
```

Comment:
this seems like it should be internal to store?

Reply:
yeah agreed